modin/sql/connection.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from ..pandas import Series, DataFrame
+
+
+class Connection(object):
+
+    def __init__(self, name):
+        self._name = name
+        self._cursor = None
+
+    def cursor(self):
+        self._cursor = Cursor()
+        return self._cursor
+
+    def commit(self):
+        pass
+
+    def close(self):
+        self._cursor = None
+
+
+class Cursor(object):
+
+    def __init__(self):
+        self._tables = {}
+
+    def execute(self, query):
+        split_query = query.split(" ")
+        if " ".join(split_query[:2]) == "CREATE TABLE":
+            column_names = " ".join(split_query[3:])\
+                .replace("(", "").replace(")", "").split(", ")
+            columns = Series(column_names)
+            self._tables[split_query[2]] = DataFrame(columns=columns)
+
+        elif " ".join(split_query[:2]) == "INSERT INTO":
```

Comment:
Can we split these into helper methods? This will make development on further functionality better in the future.

Reply:
Sure makes sense. Most likely this code will get thrown away in the future, but the skeleton might stay.